### PR TITLE
Ohio "oh" to Pupa

### PR DESCRIPTION
**Pull request by In-vincible**
- [X] people 
- [x] bills
- [x] events
- [x] votes

### CODE REVIEW

**Comment by In-vincible** [comment]
@jmcarp @mileswwatkins in case you guys have time, do review this PR :)

File: billy_metadata/oh.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+def session_list():
+    from billy.scrape.utils import url_xpath
+    sessions = url_xpath('http://archives.legislature.state.oh.us',
+                     '//form[@action="bill_search.cfm"]//input[@type="radio" and @name="SESSION"]/@value')
+    # Archive does not include current session
+    sessions.append('131')
+    return sessions
+
+
+def extract_text(doc, data):
```
Comment (jmcarp) [inline billy_metadata/oh.py]: Looks like this can be deleted.

File: billy_metadata/oh.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        '132': { 'display_name': '132st Legislature (2017-2018)',
+                '_scraped_name': '132',
+               },
+    },
+    feature_flags=['influenceexplorer', 'events'],
+    _ignored_scraped_sessions=['127', '126', '125', '124', '123', '122']
+
+)
+
+def session_list():
```
Comment (jmcarp) [inline billy_metadata/oh.py]: Can this be moved to `get_session_list` on the `Ohio` class?

File: billy_metadata/oh.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import datetime
+import lxml.html
```
Comment (jmcarp) [inline billy_metadata/oh.py]: Some of these imports shouldn't be needed after moving or deleting functions below.

File: openstates/oh/bills.py [hunk 4]
```diff
@@ -2,98 +2,99 @@
...
 
 import xlrd
 import scrapelib
 import lxml.html
+import pytz
 
 
-class OHBillScraper(BillScraper):
+class OHBillScraper(Scraper):
     jurisdiction = 'oh'
```
Comment (jmcarp) [inline openstates/oh/bills.py]: `jurisdiction` class variables can be deleted here and in other files.

File: openstates/oh/people.py [hunk 5]
```diff
@@ -116,15 +121,20 @@ def scrape_homepage(self, leg, chamber, homepage, term):
...
-
-    def scrape_page(self, chamber, term, url):
+            org = Organization(
+                name=entry,
+                chamber=chmbr,
+                classification='committee',
+            )
+            leg.add_membership(org)
+            yield org
+            # leg.add_role('committee member',
```
Comment (jmcarp) [inline openstates/oh/people.py]: Can this be deleted?

**Comment by jmcarp** [comment]
Similar comment to the AR pull request: can you add to `flake8.sh` and check for flake8 warnings? 😄

File: openstates/oh/people.py [hunk 6]
```diff
@@ -167,15 +177,16 @@ def scrape_page(self, chamber, term, url):
...
-            leg.add_office('capitol', 'Capitol Office',
-                           address=office,
-                           phone=phone,
-                           email=email)
+            leg.add_contact_detail(type='address', value=office, note='Capitol Office')
+            leg.add_contact_detail(type='voice', value=phone, note='Capitol Office')
+            leg.add_contact_detail(type='email', value=email, note='Capitol Office')
 
-            self.scrape_homepage(leg, chamber, homepage, term)
+            self.scrape_homepage(leg, chamber, homepage)
```
Comment (jmcarp) [inline openstates/oh/people.py]: It looks like `scrape_homepage` is now a generator since it yields `Organization` objects, so we need to `yield from` it.

**Comment by In-vincible** [comment]
@jamesturk @jmcarp I think I have made all the required changes.

**Comment by In-vincible** [comment]
There was a conflict in openstates/utils/__init__.py as I also edited it earlier for using "url_xpath" function from utils,

**Pull request merged**